charts/internal/shoot-cert-management-seed/templates/deployment.yaml

```diff
@@ -9,7 +9,9 @@ metadata:
     app.kubernetes.io/name: {{ include "cert-management.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+    {{- if not .Values.internalDeployment }}
```

Comment:
Can you elaborate why all of those `internalDeployment` branches are needed when there is a separate `internal-cert-management` chart?

Reply:
The files in the folder `charts/internal/internal-cert-management/templates` are copied from the other folders as needed during `make generate` to avoid duplicate maintenance.